src/epcpm/cantoxlsx.py

```diff
@@ -207,7 +208,6 @@ def gen(self) -> openpyxl.workbook.workbook.Workbook:
                 parameter_uuid_finder=self.parameter_uuid_finder,
                 pmvs_uuid_to_value_list=self.pmvs_uuid_to_value_list,
             ).gen()
-
```

Comment:
Add this line back in. Nothing changed here.

Reply:
https://github.com/epcpower/pm/pull/87/commits/d884709d1f30a2567368481b10b88d03e8de308f